### PR TITLE
Add endpoints docs to wiki

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

This PR moved the endpoint docs from the introduction docs (for example https://wiki.iota.org/introduction/reference/networks/mainnet/) to the `Develop` section of the wiki.

## Links to any relevant issues

https://github.com/iotaledger/introduction-docs/issues/152

## Type of change

- Documentation Enhancement

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: iota/develop/endpoints/devnet.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- mqtt.lb-1.h.chrysalis-devnet.iota.cafe:1883
+
+These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this [specification](https://editor.swagger.io/?url=https://raw.githubusercontent.com/rufsam/protocol-rfcs/master/text/0026-rest-api/rest-api.yaml)) over TLS.
+
+## Developer Tools
+
+- [Explorer](https://explorer.iota.org/devnet).
+- [Online Faucet](https://faucet.chrysalis-devnet.iota.cafe).
+- [cli-wallet](https://github.com/iotaledger/cli-wallet).
+- [chrysalis-faucet Code (nodejs + svelte)](https://github.com/iotaledger/chrysalis-faucet).
```
Comment (Dr-Electron) [inline iota/develop/endpoints/devnet.md:43]: As this are endpoints docs and we already mention the cli-wallet under tools in the `Develop` section I think we can remove this.
```suggestion
```

File: iota/develop/sidebars.ts [hunk 2]
```diff
@@ -101,6 +101,23 @@ module.exports = {
         },
       ],
     },
+    {
+      type: 'category',
+      label: 'Endpoints',
```
Comment (Dr-Electron) [inline iota/develop/sidebars.ts:106]: As we also mention websites like the explorer and faucet, would something like `Endpoints & Developer Infrastructure` fit better? Any suggestions? Or should we just leave it?

File: next/develop/endpoints/testnet.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## Developer Tools
+
+- [Explorer](https://explorer.shimmer.network)
+- [Online Faucet](https://faucet.testnet.shimmer.network)
+- Faucet Info API: https://faucet.testnet.shimmer.network/api/info
+- Faucet Enqueue API: https://faucet.testnet.shimmer.network/api/enqueue
+- [Cli-Wallet](https://github.com/iotaledger/cli-wallet/tree/develop)
+- [shimmer-faucet code (nodejs + svelte)](https://github.com/iotaledger/chrysalis-faucet/tree/hornet)
+- [shimmer-faucet backend](https://github.com/iotaledger/inx-faucet)
```
Comment (Dr-Electron) [inline next/develop/endpoints/testnet.md:37]: Same as above

```suggestion
```

File: shimmer/develop/endpoints/testnet.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## Developer Tools
+
+- [Explorer](https://explorer.shimmer.network)
+- [Online Faucet](https://faucet.testnet.shimmer.network)
+- Faucet Info API: https://faucet.testnet.shimmer.network/api/info
+- Faucet Enqueue API: https://faucet.testnet.shimmer.network/api/enqueue
+- [Cli-Wallet](https://github.com/iotaledger/cli-wallet/tree/develop)
+- [shimmer-faucet code (nodejs + svelte)](https://github.com/iotaledger/chrysalis-faucet/tree/hornet)
+- [shimmer-faucet backend](https://github.com/iotaledger/inx-faucet)
```
Comment (Dr-Electron) [inline shimmer/develop/endpoints/testnet.md:37]: Same as above

```suggestion
```

File: iota/develop/endpoints/devnet.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+- reference
+- Endpoints
+---
+# Devnet
+
+Since the Chrysalis update, the `testnet` is now called `devnet`.  We recommend using the `devnet` to develop and test your application.  
+
+## Public Infrastructure
+
+IOTA currently provides a load-balanced public devNet endpoint:
```
Comment (Dr-Electron) [inline iota/develop/endpoints/devnet.md:18]:
```suggestion
IOTA currently provides a load-balanced public devnet endpoint:
```

File: iota/develop/endpoints/mainnet.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+- Endpoints
+---
+# Mainnet
+
+## Public Infrastructure
+
+IOTA currently provides two load-balanced public mainnet endpoints:
+
+- chrysalis-nodes.iota.org.
+- chrysalis-nodes.iota.cafe.
```
Comment (Dr-Electron) [inline iota/develop/endpoints/mainnet.md:20]:
```suggestion
- https://chrysalis-nodes.iota.org.
- https://chrysalis-nodes.iota.cafe.
```

File: next/develop/endpoints/testnet.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+---
+description: The IOTA foundation provides load-balanced public Shimmer Beta endpoints, where MQTT and the HTTP REST API are enabled.
```
Comment (Dr-Electron) [inline next/develop/endpoints/testnet.md:2]:
```suggestion
description: The IOTA foundation provides load-balanced public testnet endpoints, where MQTT and the HTTP REST API are enabled.
```

File: next/develop/endpoints/testnet.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+- devnet
+- load-balanced
+- HTTP REST API
+- MQTT
+- reference
+- Endpoints
+---
+# Shimmer Beta
+
+Shimmer Beta is a pre-release of the Shimmer network that is currently under development.
```
Comment (Dr-Electron) [inline next/develop/endpoints/testnet.md:14]:
```suggestion
# Testnet

Testnet is a pre-release of the Shimmer network that is currently under development.
```

File: next/develop/endpoints/testnet.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+## Public Infrastructure
+
+We currently provide load-balanced public Shimmer Beta endpoints:
```
Comment (Dr-Electron) [inline next/develop/endpoints/testnet.md:18]:
```suggestion
We currently provide load-balanced public testnet endpoints:
```

File: iota/develop/sidebars.ts (hunk 2 above)
Comment (lucas-tortora) [reply iota/develop/sidebars.ts:106]: How about `Endpoints and Developer Tools` ?

File: iota/develop/endpoints/mainnet.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+# Mainnet
+
+## Public Infrastructure
+
+IOTA currently provides two load-balanced public mainnet endpoints:
+
+- https://chrysalis-nodes.iota.org.
+- https://chrysalis-nodes.iota.cafe.
+
+These endpoints have MQTT (via WebSockets) exposed and offer the HTTP REST API (according to this [specification](https://editor.swagger.io/?url=https://raw.githubusercontent.com/rufsam/protocol-rfcs/master/text/0026-rest-api/0026-rest-api.yaml))
```
Comment (jlvandenhout) [inline iota/develop/endpoints/mainnet.md:23]: Should we point to the Wiki spec now?

```suggestion
These endpoints have MQTT (via WebSockets) exposed and offer the HTTP REST API (according to this [specification](nodes/api_reference.md))
```

File: iota/develop/endpoints/devnet.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+:::
+
+We also provide single node endpoints that expose native [MQTT](https://mqtt.org/):
+
+- [https://api.thin-hornet-0.h.chrysalis-devnet.iota.cafe](https://api.thin-hornet-0.h.chrysalis-devnet.iota.cafe)
+- [https://api.thin-hornet-1.h.chrysalis-devnet.iota.cafe](https://api.thin-hornet-1.h.chrysalis-devnet.iota.cafe)
+- mqtt.lb-0.h.chrysalis-devnet.iota.cafe:1883
+- mqtt.lb-1.h.chrysalis-devnet.iota.cafe:1883
+
+These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this [specification](https://editor.swagger.io/?url=https://raw.githubusercontent.com/rufsam/protocol-rfcs/master/text/0026-rest-api/rest-api.yaml)) over TLS.
```
Comment (jlvandenhout) [inline iota/develop/endpoints/devnet.md:37]: Should we point to the Wiki spec now?

```suggestion
These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this [specification](nodes/api_reference.md)) over TLS.
```

**Review by jlvandenhout — CHANGES_REQUESTED** [review]

File: iota/develop/endpoints/mainnet.md (hunk 10 above)
Comment (jlvandenhout) [reply iota/develop/endpoints/mainnet.md:23]: Or maybe this is better, saves one click for the user.

```suggestion
These endpoints have MQTT (via WebSockets) exposed and offer the HTTP REST API (according to this [specification](nodes/rest-api/iota-rest-api.info.mdx))
```

File: iota/develop/endpoints/devnet.md (hunk 11 above)
Comment (jlvandenhout) [reply iota/develop/endpoints/devnet.md:37]: Same here, maybe this is better to save one click.

```suggestion
These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this [specification](nodes/rest-api/iota-rest-api.info.mdx)) over TLS.
```

File: iota/develop/sidebars.ts (hunk 2 above)
Comment (Dr-Electron) [reply iota/develop/sidebars.ts:106]: But I want to get rid of that name 😅 . The pages does't really contain tools. And we already have a "Tools" section in the sidebar

**Comment by lucas-tortora** [comment]
I think we may want to use the same doc except for the actual endpoints for all versions.  At the moment they are quite similar, but in one we say "The IOTA Foundation currently provides these public load-balanced endpoints", in another "We provide" and in another "IOTA provides". They all basically say the same, but it'd be nice if the only thing that changed was the network name, URLs, and available tools. WDYT?

**Comment by Dr-Electron** [comment]
> I think we may want to use the same doc except for the actual endpoints for all versions. At the moment they are quite similar, but in one we say "The IOTA Foundation currently provides these public load-balanced endpoints", in another "We provide" and in another "IOTA provides". They all basically say the same, but it'd be nice if the only thing that changed was the network name, URLs, and available tools. WDYT?

Makes sense. Do you want to change that, or should I?

**Comment by lucas-tortora** [comment]
> Makes sense. Do you want to change that, or should I?

I think it may be easier to address in a separate PR, so we can integrate this one a bit quicker seeing that it's important info.

File: shimmer/develop/endpoints/testnet.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+  - Endpoints
+---
+
+# Testnet
+
+Testnet is a pre-release of the Shimmer network that is currently under development.
+
+## Public Infrastructure
+
+We currently provide load-balanced public testnet endpoints:
```
Comment (lucas-tortora) [inline shimmer/develop/endpoints/testnet.md:19]:
```suggestion
The IOTA Foundation currently provides the following public load-balanced testnet endpoints:
```

File: iota/develop/endpoints/devnet.md [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+  - Endpoints
+---
+
+# Devnet
+
+Since the Chrysalis update, the `testnet` is now called `devnet`. We recommend using the `devnet` to develop and test your application.
+
+## Public Infrastructure
+
+IOTA currently provides a load-balanced public devnet endpoint:
```
Comment (lucas-tortora) [inline iota/develop/endpoints/devnet.md:19]:
```suggestion
The IOTA Foundation currently provides the following public load-balanced devnet endpoints:
```

File: iota/develop/endpoints/devnet.md (hunk 1 above)
Comment (jlvandenhout) [reply iota/develop/endpoints/devnet.md]: Agreed

File: iota/develop/sidebars.ts (hunk 2 above)
Comment (jlvandenhout) [reply iota/develop/sidebars.ts:106]: I'd remove the whole Developer Tools section and move faucet endpoint up right after the MQTT endpoints. We need a bit of explanation that this specific net provides a faucet to be able to get tokens to work with, provide the endpoint and then point to how to actually use the endpoint.

Comment (jlvandenhout) [reply iota/develop/sidebars.ts:106]: And move the rest of the tools in Developer Tools to the Tools section already provided in the sidebar

**Review by jlvandenhout — CHANGES_REQUESTED** [review]

**Review by lucas-tortora — APPROVED** [review]

File: next/develop/endpoints/shimmer.md [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+The IOTA Foundation currently provides the following public load-balanced Shimmer endpoints:
+
+Node API: https://api.shimmer.network  
+Example info endpoint: https://api.shimmer.network/api/core/v2/info  
+Available routes: https://api.shimmer.network/api/routes  
+Health endpoint: https://api.shimmer.network/health  
+MQTT: wss://api.shimmer.network:443/api/mqtt/v1  
+Chronicle API: https://chronicle.shimmer.network
+
+These endpoints have MQTT (via WebSockets) exposed and offer the HTTP REST API (according to the specifications [TIP-25](https://github.com/iotaledger/tips/blob/main/tips/TIP-0025/tip-0025.md), [TIP-26](https://github.com/iotaledger/tips/blob/main/tips/TIP-0026/tip-0026.md) and [TIP-28](https://github.com/iotaledger/tips/blob/main/tips/TIP-0028/tip-0028.md)) over TLS
```
Comment (Dr-Electron) [inline next/develop/endpoints/shimmer.md:27]:
```suggestion
These endpoints have MQTT (via WebSockets) exposed and offer the HTTP REST API (according to the specifications [TIP-25](/tips/tips/TIP-0025), [TIP-26](/tips/tips/TIP-0026) and [TIP-28](/tips/tips/TIP-0028)) over TLS
```

File: shimmer/develop/endpoints/shimmer.md [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+The IOTA Foundation currently provides the following public load-balanced Shimmer endpoints:
+
+Node API: https://api.shimmer.network  
+Example info endpoint: https://api.shimmer.network/api/core/v2/info  
+Available routes: https://api.shimmer.network/api/routes  
+Health endpoint: https://api.shimmer.network/health  
+MQTT: wss://api.shimmer.network:443/api/mqtt/v1  
+Chronicle API: https://chronicle.shimmer.network
+
+These endpoints have MQTT (via WebSockets) exposed and offer the HTTP REST API (according to the specifications [TIP-25](https://github.com/iotaledger/tips/blob/main/tips/TIP-0025/tip-0025.md), [TIP-26](https://github.com/iotaledger/tips/blob/main/tips/TIP-0026/tip-0026.md) and [TIP-28](https://github.com/iotaledger/tips/blob/main/tips/TIP-0028/tip-0028.md)) over TLS
```
Comment (Dr-Electron) [inline shimmer/develop/endpoints/shimmer.md:27]:
```suggestion
These endpoints have MQTT (via WebSockets) exposed and offer the HTTP REST API (according to the specifications [TIP-25](/tips/tips/TIP-0025), [TIP-26](/tips/tips/TIP-0026) and [TIP-28](/tips/tips/TIP-0028)) over TLS
```

File: next/develop/endpoints/testnet.md [hunk 16]
```diff
@@ -0,0 +1,30 @@
...
+- Node API: https://api.testnet.shimmer.network
+  - Example info endpoint: https://api.testnet.shimmer.network/api/core/v2/info
+  - Available routes: https://api.testnet.shimmer.network/api/routes
+  - Health endpoint: https://api.testnet.shimmer.network/health
+- MQTT: wss://api.testnet.shimmer.network:443/api/mqtt/v1
+- Faucet Info API: https://faucet.testnet.shimmer.network/api/info
+- Faucet Enqueue API: https://faucet.testnet.shimmer.network/api/enqueue
+- Chronicle API: https://chronicle.testnet.shimmer.network
+
+These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this specifications [TIP-25](https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/main/tips/TIP-0025/core-rest-api.yaml), [TIP-26](https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/main/tips/TIP-0026/indexer-rest-api.yaml) over TLS.
```
Comment (jlvandenhout) [inline next/develop/endpoints/testnet.md:30]:
```suggestion
These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this specifications [TIP-25](nodes/rest-api/iota-rest-api.info.mdx), [TIP-26](https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/main/tips/TIP-0026/indexer-rest-api.yaml) over TLS.
```

This one too probably?

File: shimmer/develop/endpoints/testnet.md [hunk 17]
```diff
@@ -0,0 +1,30 @@
...
+- Node API: https://api.testnet.shimmer.network
+  - Example info endpoint: https://api.testnet.shimmer.network/api/core/v2/info
+  - Available routes: https://api.testnet.shimmer.network/api/routes
+  - Health endpoint: https://api.testnet.shimmer.network/health
+- MQTT: wss://api.testnet.shimmer.network:443/api/mqtt/v1
+- Faucet Info API: https://faucet.testnet.shimmer.network/api/info
+- Faucet Enqueue API: https://faucet.testnet.shimmer.network/api/enqueue
+- Chronicle API: https://chronicle.testnet.shimmer.network
+
+These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this specifications [TIP-25](https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/main/tips/TIP-0025/core-rest-api.yaml), [TIP-26](https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/main/tips/TIP-0026/indexer-rest-api.yaml) over TLS.
```
Comment (jlvandenhout) [inline shimmer/develop/endpoints/testnet.md:30]:
```suggestion
These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this specifications [TIP-25](nodes/rest-api/iota-rest-api.info.mdx), [TIP-26](https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/main/tips/TIP-0026/indexer-rest-api.yaml) over TLS.
```

and this one.

**Review by jlvandenhout — CHANGES_REQUESTED** [review]

File: next/develop/sidebars.ts [hunk 18]
```diff
@@ -180,6 +180,23 @@ module.exports = {
...
+      items: [
+        {
+          type: 'doc',
+          id: 'endpoints/shimmer',
+          label: 'Mainnet',
+        },
+        {
+          type: 'doc',
+          id: 'endpoints/testnet',
+          label: 'Devnet',
```
Comment (Dr-Electron) [inline next/develop/sidebars.ts:196]:
```suggestion
          label: 'Shimmer',
        },
        {
          type: 'doc',
          id: 'endpoints/testnet',
          label: 'Testnet',
```

File: shimmer/develop/sidebars.ts [hunk 19]
```diff
@@ -180,6 +180,23 @@ module.exports = {
...
+      items: [
+        {
+          type: 'doc',
+          id: 'endpoints/shimmer',
+          label: 'Mainnet',
+        },
+        {
+          type: 'doc',
+          id: 'endpoints/testnet',
+          label: 'Devnet',
```
Comment (Dr-Electron) [inline shimmer/develop/sidebars.ts:196]:
```suggestion
          label: 'Shimmer',
        },
        {
          type: 'doc',
          id: 'endpoints/testnet',
          label: 'Testnet',
```

File: shimmer/develop/endpoints/testnet.md [hunk 20]
```diff
@@ -0,0 +1,30 @@
...
+- Node API: https://api.testnet.shimmer.network
+  - Example info endpoint: https://api.testnet.shimmer.network/api/core/v2/info
+  - Available routes: https://api.testnet.shimmer.network/api/routes
+  - Health endpoint: https://api.testnet.shimmer.network/health
+- MQTT: wss://api.testnet.shimmer.network:443/api/mqtt/v1
+- Faucet Info API: https://faucet.testnet.shimmer.network/api/info
+- Faucet Enqueue API: https://faucet.testnet.shimmer.network/api/enqueue
+- Chronicle API: https://chronicle.testnet.shimmer.network
+
+These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this specifications [TIP-25](nodes/rest-api/iota-rest-api.info.mdx), [TIP-26](https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/main/tips/TIP-0026/indexer-rest-api.yaml) over TLS.
```
Comment (jlvandenhout) [inline shimmer/develop/endpoints/testnet.md:30]:
```suggestion
These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this specifications [TIP-25](nodes/core-rest-api/iota-core-rest-api.info.mdx), [TIP-26](https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/main/tips/TIP-0026/indexer-rest-api.yaml) over TLS.
```

**Review by jlvandenhout — CHANGES_REQUESTED** [review]

**Review by jlvandenhout — APPROVED** [review]

**Pull request merged**